packages/dds/task-manager/src/index.ts

```diff
@@ -3,5 +3,12 @@
  * Licensed under the MIT License.
  */
 
+/**
+ * Contains a distributed data structure, {@link ITaskManager}, to track the queues of clients that want to
```

Comment:
@scottn12 Is there any more high-level information that would be useful to add here? Or do you think linking to the `ITaskManager` interface is sufficient?

Reply:
I think the link should be sufficient 🙂